test/src/qa/hpcp/test_hpcp_parameters.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+
+# Tuple of probe signals (tones and noise) for testing
+testSignals = ('Tones A0 - G9', 
+        'Tones A0 - G1', 
+        'Tones A1 - G2', 
+        'Tones A2 - G3', 
+        'Tones A3 - G4', 
+        'Tones A4 - G5', 
+        'Tones A5 - G6', 
+        'Tones A6 - G7', 
+        'Tones A7 - G8', 
+        'Tones A8 - G9', 
+        'White Noise', 
+        'LPF WN midi ref. 0', 
+        'LPF WN midi ref. 1', 
+        'LPF WN midi ref. 2', 
+        'LPF WN midi ref. 3', 
+        'LPF WN midi ref. 4', 
+        'LPF WN midi ref. 5', 
+        'LPF WN midi ref. 6', 
+        'LPF WN midi ref. 7', 
+        'LPF WN midi ref. 8', 
+        'LPF WN midi ref. 9', 
+        'LPF WN midi ref. 10', 
+        'LPF WN midi ref. 11', 
+        'BPF WN midi ref. 19', 
+        'BPF WN midi ref. 20', 
+        'BPF WN midi ref. 21', 
+        'BPF WN midi ref. 22', 
+        'BPF WN midi ref. 23', 
+        'BPF WN midi ref. 24', 
+        'BPF WN midi ref. 25', 
+        'BPF WN midi ref. 26', 
+        'BPF WN midi ref. 27', 
+        'BPF WN midi ref. 28', 
+        'BPF WN midi ref. 29', 
+        'BPF WN midi ref. 30', 
+                 )
+# Test titles
+testTitles = {
+    'testWindowWholeRange': 'HPCP and window type in different scales',
+    'testFrameSize': 'HPCP, window type and frame size in the lower scales', 
+    'testMaxFreq': 'HPCP, window type and max frequency in the upper octaves',
+    'testJordiNL': 'Jordi non-linear filtering',
+    'testWeigth': 'Weigthing function and window type',
+    'testLPFilterMeanSpectrum': 'Mean spectrum of low pass filtered white noise',
+    'testBPFilterMeanSpectrum': 'Mean spectrum of band pass filtered white noise',
+    'testLPFilterFrameSpeaks': 'Frame spectral peaks of low pass filtered white noise',
+    'testBPFilterFrameSpeaks': 'Frame spectral peaks of band pass filtered white noise',
+    'testNormalizeWindow': 'Window normalization for different window types',
+    'testNormalizeHPCP': 'HPCP normalization',
+}
+
+# Mapping between the selected test and the probe 
+test2SignalsMapping = {'testWindowWholeRange': testSignals[0:10],
```

Comment:
It may have more sense to hardcode signal names here in the list (['Tones A0 - G9', 'Tones A0 - G1', 'Tones A1 - G2', 'Tones A2 - G3', ...]) . It will occupy more space but this way, no indexes are required, and it'll be more easy to see what signals are actually run for each test and expand or edit the list of tests when necessary.